test/kubemark/start-kubemark.sh

```diff
@@ -394,6 +416,13 @@ current-context: kubemark-context")
     sed -i'' -e "s/{{kubemark_autoscaler_max_nodes}}/${KUBEMARK_AUTOSCALER_MAX_NODES}/g" "${RESOURCE_DIRECTORY}/addons/cluster-autoscaler.json"
   fi
 
+  # Kube DNS.
+  if [[ "${ENABLE_KUBEMARK_KUBE_DNS}" == "true" ]]; then
+    echo "Setting up kube-dns"
+    KUBE_DNS_DOMAIN="${KUBE_DNS_DOMAIN:-cluster.local}"
```

Comment:
I think you should expose this variable in `cluster/kubemark/gce/config-default.sh` to make it useful. Right now, you're just defining and using it in the next line.

Reply:
done